testing/tests/DevExpress.viz.charts/chart.integration.tests.js

```diff
@@ -2282,9 +2282,103 @@ QUnit.test('Has no exception when hiding point markers automatically (both hidin
     });
 
     assert.notOk(chart.getAllSeries()[0].getVisiblePoints()[0].graphic); // area algorithm
-    assert.notOk(chart.getAllSeries()[1].getVisiblePoints()[0].graphic); // intersection algorithm
 });
 
+QUnit.test('All points should be hidden when all points are overlapped', function(assert) {
+    const dataSource = [ {
+        country: 'USA',
+        val1: 10,
+        val2: 11,
+        val3: 10.5,
+        val4: 40,
+        val5: 50
+    },
```

Comment:
u can write it as `}, {` to make code more compact

Reply:
Fixed